src/VisualStudio/Core/Def/Progression/GraphQueries/CallsGraphQuery.cs

```diff
@@ -18,6 +19,8 @@ internal sealed class CallsGraphQuery : IGraphQuery
 {
     public async Task<GraphBuilder> GetGraphAsync(Solution solution, IGraphContext context, CancellationToken cancellationToken)
     {
+        using var _ = Logger.LogBlock(FunctionId.GraphQuery_Calls, KeyValueLogMessage.Create(LogType.UserAction), cancellationToken);
```

Comment:
do you need to do something to get this telemetry to actually fire?  i thought Logger.Log didn't work that way automatically?

Reply:
KeyValueLogMessage.Create defaults logLevel to LogLevel.Information
TelemetryLogger.IgnoreMessage checks logMessage.LogLevel < LogLevel.Information

Verified telemetry was being sent by using vs telemetry monitor.